pages/contact.html

```diff
@@ -41,5 +41,22 @@
     </a>
 </div>
 </div>
+
+<div class="ui main text container">
+    <h1 class="ui dividing header" style="padding-top:8px">School Contact Details</h1>
+    <p>Email School: <a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
+    <p>Web: mountsioncbssecondary.ie</p>
+    <p>Telephone Number: 051 377378</p>
+    <p>Fax Number: 051-376468</p>
+    <p>School Roll No. 64930i</p>
+    <p>Address:</p>
+    <p>Mount Sion CBS Secondary School,</p>
+    <p>Barrack Street,</p>
+    <p>Waterford City.</p>
+    <h2 class="ui header">School Location</h2>
```

Comment:
If you are going with`dividing` headers, try to keep all headers the same, we can revisit the style we want to follow later, but for now, we should try stick with one.